Neos.Neos/Classes/FrontendRouting/NodeAddressFactory.php

```diff
@@ -79,6 +80,7 @@ public function createFromUriString(string $serializedNodeAddress): NodeAddress
         $dimensionSpacePoint = DimensionSpacePoint::fromUriRepresentation($dimensionSpacePointSerialized);
         $nodeAggregateId = NodeAggregateId::fromString($nodeAggregateIdSerialized);
 
+        // hahah funny und nun?
```

Comment:
hmm this is not easy to solve without breaking things.

The factory needs at construction already the correct cr id, so we cannot read it from the uri, because this is a non static method - and if we wanted to we need the cr registry ...